ecs_model_deployer/src/lib/schema.ts

```diff
@@ -618,6 +618,7 @@ const RawConfigSchema = z
                 instanceProfilePrefix: z.string().optional(),
             })
             .optional(),
+        subnetIds: z.array(z.string()).optional(),
```

Comment:
I would make this more explicit: `subnetIds: z.array(z.string().startsWith('subnet-')).optional()`

Reply:
Thinking about this some more, we should force `vpcId` to be required if this exists. We can do this with https://github.com/colinhacks/zod#refine as we've done with other conditional properties.